zbl/src/capture/mod.rs

```diff
@@ -68,7 +69,8 @@ impl Capture {
         )?;
 
         let session = frame_pool.CreateCaptureSession(&capture_item)?;
-        session.SetIsCursorCaptureEnabled(capture_cursor)?;
+        session.SetIsCursorCaptureEnabled(is_cursor_capture_enabled)?;
+        session.SetIsBorderRequired(is_border_required)?;
```

Comment:
Let's account for the possibility of this not being supported with a meaningful error message. Something like this should work:

```suggestion
        if !is_border_required {
            if let Err(e) = session.SetIsBorderRequired(is_border_required) {
                log::warn!(
                    "got '{}' when trying to disable the capture border - see https://github.com/modelflat/zbl/pull/4 for more info",
                    e
                );
            }
        }
```